.coderabbit.yaml

```
@@ -0,0 +1,59 @@
+  4. 음성 파일 업로드 - 파일 크기/형식 검증
+  5. 사용자 인증/권한 - 보안 취약점 확인
+  6. 실시간 피드백 - WebSocket 연결 안정성
+
+knowledge_base:
+  learnings:
+    enabled: true
+    scope: "repository"
+
+# 자동 요약 설정
+auto_reply: false
+enable_free_tier: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)